src/mopeka.c

```diff
@@ -159,18 +164,28 @@ static int mopeka_xlate_level(struct VeItem *root, VeVariant *val, uint64_t rv)
 	temp = veItemValueInt(root, "Temperature");
 	temp += 40;
 
+	// Check for presence of extension bit on certain hardware/firmware -
+	// it will always be 0 on old firmware/hardware where raw value saturates
+	// at 16383.  When extension bit is set, the raw_value resolution changes
+	// to 4us with 16384us offet.  Thus legacy sensors and firmware still 
+	// have 0 to 16383us range with 1us, and new versions add the range 16384us 
```

Comment:
I hope they leave this comment in, it's a detail that's easy to miss, but I don't have any control over that.